wagtail/admin/static_src/wagtailadmin/scss/layouts/login.scss

```diff
@@ -82,7 +82,12 @@ input[type=checkbox]:before {
         padding: 0;
 
         label {
-            display: none;
+            position: absolute;
+            left: -10000px;
+            top: auto;
+            width: 1px;
+            height: 1px;
+            overflow: hidden;
```

Comment:
Just wondering if the mixin `visuallyhidden` would be best to use here, to avoid duplication?

See definition - client/scss/tools/_mixins.general.scss

Reply:
I did not realized there is a mixin for it already. Thank you for the heads up. I just added the commit with visuallyhidden change.